src/timeAgoInWords.test.ts

```diff
@@ -70,6 +92,112 @@ describe('timeAgoInWords', () => {
 		).toBe('Friday 15 Nov 2019');
 	});
 
+	it('does not pluralise the unit when the delta is one', () => {
+		const onHourAgo = new Date(Date.UTC(2019, 10, 17, 11, 0, 0)).getTime();
```

Comment:
I think there’s a small typo here

```suggestion
const oneHourAgo = new Date(Date.UTC(2019, 10, 17, 11, 0, 0)).getTime();
```